lib/Utils/Feature.php

```diff
@@ -8,24 +8,63 @@ class Feature
     private static $initialFile = 'functions.php';
     private static $features = [];
 
+    /**
+     * Gets a feature options.
```

Comment:
Gets all options for a feature.

Returns all parameters passed to `add_theme_support` in the lib/Init.php file.